packages/keycloak-user-management/src/components/UserGroupDetailView/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { Col, Space, Typography, Spin } from 'antd';
+import { CloseOutlined } from '@ant-design/icons';
+import { useHistory } from 'react-router';
+import { Resource404 } from '@opensrp/react-utils';
+import { sendErrorNotification } from '@opensrp/notifications';
+import { Button } from 'antd';
+import { URL_USER_EDIT, URL_USER_GROUPS } from '../../constants';
+import { UserGroupMembers } from '../UserGroupsList';
+import { KeycloakUserGroup } from 'keycloak-user-management/src/ducks/userGroups';
+import { Link } from 'react-router-dom';
+import { loadGroupDetails, loadGroupMembers } from '../UserGroupsList/utils';
+import { ERROR_OCCURED, MEMBERS, NAME, ROLES } from '../../lang';
+const { Text } = Typography;
+
+/** typings for the view details component */
+export interface ViewDetailsProps {
+  groupId: string;
+  keycloakBaseURL: string;
+}
+export const defaultProps = {
+  groupId: '',
+};
```

Comment:
for the props that are required (not optional) in interface are already enforced tobe passed down the component that is why i found default props to be used for that is something that is unnecessary

Reply:
agreed will remove default props